src/MongoDB/Cursor.c

```diff
@@ -62,21 +62,6 @@ static int php_phongo_cursor_valid(php_phongo_cursor_t* cursor) /* {{{ */
 	return FAILURE;
 } /* }}} */
 
-static void php_phongo_cursor_get_current_key(php_phongo_cursor_t* cursor, zval* key) /* {{{ */
```

Comment:
I just realized `php_phongo_cursor_valid` is also only called once from `valid()`. Do you want to include that in a separate commit? If not, we can get to it down the line at some point.

If so, I'd also suggest just removing the "MongoDB\Driver\Cursor iterator handlers" comment (and the corresponding `}}}`) since we really don't have iterator handlers anymore.